### PR TITLE
4.0] pre-updatecheck rtl fixes 

### PR DESCRIPTION
**Pull request by brianteeman**
php and css changes for the joomla update component pre-update check tab to correctly align the caption and the title

### LTR before and after
image

### RTL before
image

### RTL after
image

### CODE REVIEW

File: administrator/language/en-GB/langmetadata.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
 		<name>English (United Kingdom)</name>
 		<nativeName>English (United Kingdom)</nativeName>
 		<tag>en-GB</tag>
-		<rtl>0</rtl>
+		<rtl>1</rtl>
```
Comment (wilsonge) [inline administrator/language/en-GB/langmetadata.xml]: Assume this is here for testing or just an accidental commit?

Comment (brianteeman) [reply administrator/language/en-GB/langmetadata.xml]: oops

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 3352f6be5abe1e8d90d8acb4204eb6445145b69b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34857">issues.joomla.org/tracker/joomla-cms/34857</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thank you

**Pull request merged**